storage/innobase/xtrabackup/src/backup_mysql.h

```diff
@@ -245,6 +245,11 @@ bool is_fts_index(const std::string &tablespace);
 @return true if tablespace is temporary table. */
 bool is_tmp_table(const std::string &tablespace);
 
+/** Identifies if table is blocked by select - PS compression dictionary.
```

Comment:
what is this 'blocking by select'?

Reply:
```
mysql> SELECT * FROM mysql.compression_dictionary;
ERROR 3554 (HY000): Access to system table 'mysql.compression_dictionary' is rejected.
mysql>
```

Reply:
would be nice to add that SELECT from mysql.compression_* tables is not allowed by Server.